Separate C7 and C8 Rules

philippfromme opened the pull request:
Requires https://github.com/bpmn-io/bpmnlint/pull/99.

nikku commented on package.json:
```diff
@@ -25,7 +25,6 @@
   "license": "MIT",
   "devDependencies": {
     "bpmn-moddle": "^7.1.3",
-    "bpmnlint": "^7.8.0",
```
Is it intentional to move this to a production dependency?

philippfromme replied:
Ooops!

philippfromme replied:
Fixed!

nikku commented:
Looks good. Took me a little bit to grasp what we fixed with https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/87/commits/0c675bc9a3e7f44ef28727fee04e49145a687ad7 :wink:.

philippfromme commented:
> Took me a little bit to grasp what we fixed with [0c675bc](https://github.com/camunda/bpmnlint-plugin-camunda-compat/commit/0c675bc9a3e7f44ef28727fee04e49145a687ad7) 😉.

Everything is what we fixed! But mainly the fact that we only care about executable processes and some conventions.
